changelog/README.md

```diff
@@ -1,30 +1,14 @@
 # Changelog
 
+## 13th March 2024
+* Introduced detailed guide for [Correct widget implementation without caching the file](../booking-engine-widget/use-cases/disable-widget-caching.md)
```

Comment:
As suggested elsewhere, I think the page name should be **Disabling widget caching** and that is how we should reference it.
```suggestion
* Introduced detailed guide for [Disabling widget caching](../booking-engine-widget/use-cases/disable-widget-caching.md)
```